examples/example-app-v12/src/app/demo/demo.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { jest } from '@jest/globals';
+
+import { LightswitchComponent, MasterService, ValueService, ReversePipe } from './demo';
+
+// eslint-disable-next-line jest/no-export
+export class FakeValueService extends ValueService {
+  value = 'faked service value';
+}
+
+describe('demo (no TestBed):', () => {
+  describe('ValueService', () => {
+    let service: ValueService;
+    beforeEach(() => {
+      service = new ValueService();
+    });
+
+    it('#getValue should return real value', () => {
+      expect(service.getValue()).toBe('real value');
+    });
+
+    // eslint-disable-next-line jest/no-done-callback
+    it('#getObservableValue should return value from observable', (done: jest.DoneCallback) => {
```

Comment:
`jest-circus` runner (which is default in Jest 27) doesn't have `done` callback anymore :)

You can convert the observable to `Promise` and use `async/await` to replace `done` callback. Not sure why the tests use `done` callback didn't fail. We should remove the `done` callback for other places too.

Reply:
k, will do soon

Reply:
@ahnpnl resolved